packages/cli/test/fixtures/unit/create-git-meta/no-email/config

```diff
@@ -0,0 +1,11 @@
+[core]
+    repositoryformatversion = 0
+    fileMode = false
+    bare = false
+    logallrefupdates = true
+[user]
+    name = TechBug2012
+    email = <>
+[remote "origin"]
+    url = https://github.com/MatthewStanciu/git-test
+    fetch = +refs/heads/*:refs/remotes/origin/*
```

Comment:
The tests will still work, but for consistency with real git directories, can we move this into `packages/cli/test/fixtures/unit/create-git-meta/no-email/git/config`?